_about/notes/index.md

```diff
@@ -14,22 +14,19 @@ redirect_from:
 toc: false  
 ---
 
-{% include section-index.html docs=site.about %}
```

Comment:
This needs to be present like in all other index pages in the site, so as to enable good mobile navigation. Please just let that there and remove the - and indent in the subsequent sections.